Gems/Atom/RHI/DX12/Code/Source/RHI/ImageView.cpp

```diff
@@ -61,6 +61,9 @@ namespace AZ
                     m_depthStencilReadDescriptor);
             }
 
+            m_hash = TypeHash64(viewDescriptor.GetHash(), m_hash);
+            m_hash = TypeHash64(m_format, m_hash);
```

Comment:
Besides format, bindFlags also has a source from the image, but it is not cached in ImageView. Should we take this into consideration and maybe has with Image::GetHash as well?